src/Crypt32.Shared/Crypt32.cs

```diff
@@ -51,5 +51,12 @@ public static partial class Crypt32
         [DllImport(nameof(Crypt32), SetLastError = true)]
         [return: MarshalAs(UnmanagedType.Bool)]
         private static extern bool CertCloseStore(IntPtr hCertStore, CertCloseStoreFlags dwFlags = CertCloseStoreFlags.None);
+
+        [DllImport(nameof(Crypt32), CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
No method parameter takes character/string types. So the `CharSet` parameter in your `DllImport` attribute seems superfluous.